### ISSUE TITLE
Unsafe overridden ToString on ChatCompletion

### ISSUE THREAD

**Issue by isaacabraham** [openai/openai-dotnet#152]
[This](https://github.com/openai/openai-dotnet/blob/f6e1c21a39a5e0e5c33af9f1fd267cbadd239a20/src/Custom/Chat/ChatCompletion.cs#L68) override crashes as soon as there is no content returned e.g. in the case of a FunctionTool response.

**Comment by sunnynagavo** [comment]
may be change it to this ? 

/// <summary>Returns the text of the first element in the Content collection, or an empty string if the collection is null or empty.</summary>
public override string ToString() => Content != null && Content.Count > 0 ? Content[0].Text : string.Empty;

**Comment by isaacabraham** [comment]
@joseharriaga I would agree with this. Overriding `ToString() ` is a somewhat opaque thing to do - especially if it's encapsulating some logic or varying data. Having a specific, well-known property sounds a safer and more reasonable thing to do.

`ToString()` should *never* crash - it's called implicitly in many areas of .NET applications.
